### PR TITLE
fix shell quotes on windows

### PR DESCRIPTION
**Pull request by Br4nd0R**
# Issues Closed
* Fixed and issue where `sfdx` commands that had options with spaces failed to execute on Windows.

### CODE REVIEW

File: cumulusci/core/tests/test_sfdx.py [hunk 1]
```diff
@@ -23,6 +25,15 @@ def test_windows_quoting(self, Command):
         cmd = Command.call_args[0][0]
         assert cmd == r'sfdx cmd "a\"b" -u token'
 
+    @mock.patch("platform.system", mock.Mock(return_value="Windows"))
+    @pytest.mark.skipif(
+        not sys.platform.startswith("win"),
+        reason="Special handling of shell quotes on windows",
+    )
```
Comment (davisagli) [inline cumulusci/core/tests/test_sfdx.py:32]: Since you're mocking platform.system you don't need to skip the test based on the platform, right?

**Review by davisagli — CHANGES_REQUESTED** [review]

File: cumulusci/core/tests/test_sfdx.py (hunk 1 above)
Comment (Br4nd0R) [reply cumulusci/core/tests/test_sfdx.py:32]: Ah yes. My intent was to remove the call to `mock.patch` and have this execute in the windows feature test check in GitHub. Does that sound alright to you?

Comment (davisagli) [reply cumulusci/core/tests/test_sfdx.py:32]: @Br4nd0R I'd rather go the other way and keep the mock but remove the skip. That way if we make a change that breaks it, we'll find out regardless of which platform we run the test on.

Comment (Br4nd0R) [reply cumulusci/core/tests/test_sfdx.py:31]: Sure, that works for me.

**Review by davisagli — APPROVED** [review]

**Pull request merged**